Src/library/vision2/implementation/mswin/support/ev_wel_control_container_imp.e

```diff
@@ -223,6 +224,15 @@ feature {NONE} -- WEL Implementation
 			internal_wm_size_called := True
 		end
 
+
```

Comment:
remove unwanted extra empty line before and after.

Reply:
Done